### PR TITLE
Fixed module name changes in IPython 4.0

### PR DESCRIPTION
**Pull request by brianhuey**
IPython 4.0 has split some modules into separate packages (ie. jupyter_console, traitlets), therefore ipy_repl.py throws ImportErrors.

### CODE REVIEW

File: config/Python/ipy_repl.py [hunk 1]
```diff
@@ -90,7 +101,7 @@ def send_netstring(sock, msg):
 def complete(zmq_shell, req):
     kc = kernel_client(zmq_shell)
     msg_id = kc.shell_channel.complete(**req)
-    msg = kc.shell_channel.get_msg(timeout=10)
+    msg = kc.shell_channel.get_msg(timeout=0.5)
```
Comment (MattDMo) [inline config/Python/ipy_repl.py:111]: Why did you change this to 0.5?

Comment (brianhuey) [inline config/Python/ipy_repl.py:111]: Sorry, that was unintentional.

**Comment by gacelita** [comment]
Please solve the conflicts and I'll merge it as fast as something very fast
